Calculator/Calculator/ViewController.swift

```diff
@@ -7,11 +7,151 @@
 import UIKit
 
 class ViewController: UIViewController {
+    @IBOutlet weak var currentNumberLabbel: UILabel!
+    @IBOutlet weak var currentOperatorLabel: UILabel!
+    @IBOutlet weak var operationRecord: UIStackView!
+    @IBOutlet weak var operationRecordScrollView: UIScrollView!
+    var touchedNumber: String = "0"
+    var allOperation: [String] = []
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        clearAllHistory()
+    }
+}
 
+// MARK: IBAction
+extension ViewController {
+    @IBAction func clickNumberButton(_ sender: UIButton) {
+        if currentNumberLabbel.text == "0" {
+            touchedNumber = sender.currentTitle ?? ""
+        } else {
+            touchedNumber += sender.currentTitle ?? ""
+        }
+        currentNumberLabbel.text = changeDecimalFormat(touchedNumber)
+    }
+    
+    @IBAction func clickOperatorButton(_ sender: UIButton) {
+        if touchedNumber != "0" {
+            addNumberAndOperator(currentOperatorLabel.text ?? "", touchedNumber)
+        }
+        currentOperatorLabel.text = sender.currentTitle
+    }
+    
+    @IBAction func clickDotButton(_ sender: UIButton) {
+        guard let currentNumber = currentNumberLabbel.text,
+                (currentNumber.contains(".") == false) else { return }
```

Comment:
용도가 확실하지 않은 괄호는 생략하길 권장드립니다.
```suggestion
                currentNumber.contains(".") == false else { return }
```